common/changes/@gooddata/sdk-ui-all/IMJ-RAIL-4258-Changes_2022-10-18-12-00.json

```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Improved custom attribute filters was added.  We added support for customization of the AttributeFilter and AttributeFilterButton components and new features, such as static and hidden elements.",
```

Comment:
were added

Reply:
also, there are two spaces before the second sentece

Reply:
Fixed